styles_.css

```diff
@@ -0,0 +1,1069 @@
+  }
+
+  .mylink-portfolio {
+    width: 29px;
+    height: 29px;
+  }
+
+  .mycard {
+    width: 100%;
+  }
+
+  .mycard-container {
+    display: grid;
+    grid-template-rows: auto auto auto auto;
+    background: #f7f7f9;
+    justify-content: center;
+    row-gap: 80px;
+    right: 48px;
+  }
+
+  .mycard-place-holder {
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-gap: 40px;
+    color: white;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    height: 496px;
+    position: relative;
+    width: 88%;
+    left: 5%;
+  }
+
+  .mycard-wrapper {
+    display: grid;
+    position: relative;
+    transition: transform 2s;
+  }
+
+  .mycard-wrapper-1 {
+    content: url("img/project.png");
+  }
+
+  .mycard-wrapper-2 {
+    content: url("img/project2.png");
+    display: grid;
+    grid-column: 2;
+    grid-row: 1;
+    flex-grow: 1;
+  }
+
+  .mycard-wrapper-3 {
+    content: url("img/project3.png");
+  }
+
+  .mycard-wrapper-4 {
+    content: url("img/project4.png");
+    display: grid;
+    grid-column: 2;
+    grid-row: 1;
+  }
+
+  .mycard-wrapper:hover {
+    transform: scale(1.1);
+  }
+
+  .mycard-header {
+    position: relative;
+    top: -112px;
+  }
+
+  .mycard-place-holder-2 .mycard-header {
+    left: 10%;
+  }
+
+  .mycard-place-holder-4 .mycard-header {
+    left: 10%;
+  }
+
+  .mycard-desc {
+    width: 91%;
+    font-weight: 800;
+  }
+
+  .mycard-btn {
+    /* top:20px; */
+  }
+
+  .mycard-button {
+    height: 24px;
+    width: 53px;
+  }
+
+  .mylanguage-header-title::after {
+    content: url(img/logo.png);
+    position: absolute;
+    right: 10px;
+  }
+
+  .mylanguage-header-title-1::after {
+    content: url(./img/logo.png);
+    transform: rotate(90deg);
+    position: absolute;
+    right: 10px;
+  }
+
+  .myprofile-details {
+    display: grid;
+    grid-template-columns: 30% 70%;
+    flex-wrap: wrap;
+    grid-gap: 40px;
+    margin-top: 0;
+  }
+
+  .myprofile-details-container {
+    left: 20%;
+    display: block;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+    width: 179%;
+  }
+
+  .mylanguages-place-holder {
+    width: 67%;
+    left: 13%;
+  }
+
+  .myprofile-details-header-title {
+    font-family: 'Poppins', serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    color: #091e42;
+    display: flex;
+    align-items: center;
+    margin-top: 0;
+  }
+
+  .myprofile-details-header {
+    width: 87%;
+  }
+
+  .myprofile-details-header-dsc {
+    width: 79%;
+    height: 20%;
+    top: 13%;
+  }
+
+  .myprofile-details-header-connect {
+    top: 0%;
+  }
+
+  .mylink-media {
+    top: 18%;
+    left: -6%;
+  }
+
+  .mylink-button-profile {
+    top: 11%;
+  }
+
+  .mylanguage-header-title .mylanguage-header-title-1::after {
+    content: url(img/logo.png);
+    transform: rotate(90deg);
+    position: absolute;
+    right: 0;
+  }
+
+  .mylanguage-keyfeatures-tech {
+    display: flex;
+    flex-direction: row;
+  }
+
```

Comment:
Kindly adjust the about myself section to match the design. The languages have icons within the boxes and the horizontal lines below skills and frameworks titles are faded.

Your design:
![image](https://user-images.githubusercontent.com/20554583/222940425-5d6e12d7-c6e2-4957-bf0f-e0d6d214a74d.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/222940430-4872b4bc-1d9a-42bf-84cd-04c951c49649.png)